backend/src/main/java/ca/bc/gov/app/validator/SubmissionValidatorService.java

```diff
@@ -209,8 +260,9 @@ private Mono<ClientSubmissionDto> validateSubmissionDto(ClientSubmissionDto requ
           )
       );
     }
-
-    return Mono.empty();
+    
+    return checkSubmissionLimit(request.userId())
+            .then(Mono.just(request)); 
```

Comment:
Be careful when using `then` as it will run unconditionally, even in some exception cases

Reply:
Resolved